src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Windows.cs

```diff
@@ -45,5 +47,17 @@ internal static int GetCpuUsage()
 
             return cpuUsage;
         }
+
+        internal static long GetWorkingSet()
+        {
+            Interop.ProcessMemoryCounters memoryCounters;
+            memoryCounters.cb = (uint)(Marshal.SizeOf(typeof(Interop.ProcessMemoryCounters)));
```

Comment:
Do not use `Marshal.SizeOf`. It is a slow expensive call. Use `sizeof(ProcessMemoryCounters)` instead.